README.md

```diff
@@ -59,33 +59,85 @@ cmake --install build -j $(($(nproc) * 2))
 
 ## Compositing Layered Images
 
-This fork of IceT has been extended to support layered images with more than
-one fragment (i.e. color and depth) per pixel.  Layered images are useful for
+In traditional parallel rendering (as in the original IceT), each pixel in an 
+image contains a single fragment: a color and, optionally, a depth value 
+representing the nearest visible surface or the alpha-composited result along
+the viewing ray. This works well when the domains held by the rendering processes
+can be globally sorted by depth (i.e., convex decompositions).
+
+Layered compositing extends this idea by allowing each pixel to hold multiple 
+fragments (“layers”), each representing a different volume subdomain or other 
+partially transparent object along the ray. This is crucial for
 applications that require ordered compositing, but cannot guarantee a total
-visibility ordering of all processes.  One such use case is direct volume
+visibility ordering of all processes. One such use case is direct volume
 rendering of non-convex domain decompositions.  For more information, see our
 [publication][layered-icet-paper].
 
-Currently, layered compositing is only implemented for pre-rendered images
-via the function `icetCompositeImageLayered`.  Compared to
-`icetCompositeImage`, it requires an extra argument specifying the number of
-fragments per pixel.  That number must be the same for all pixels in the
-image, but may differ between processes.  Color and depth values must be
-stored contiguously per pixel, ordered front to back.  Empty fragments with
-an alpha value of zero must be ordered after any active fragments at that
-pixel.  The individually sorted pixels are then given in the same scanline
+Layered-IceT supports parallel compositing of layered images produced by an 
+external rendering application via the `icetCompositeImageLayered` function:
+
+```c
+IceTImage icetCompositeImageLayered(const IceTVoid *color_buffer,
+                                    const IceTVoid *depth_buffer,
+                                    IceTInt num_layers,
+                                    const IceTInt *valid_pixels_viewport,
+                                    const IceTDouble *projection_matrix,
+                                    const IceTDouble *modelview_matrix,
+                                    const IceTFloat *background_color)
+```
+
+The `color_buffer` and `depth_buffer` parameters contain the linearized color and
+depth layered buffers for the current process. Fragments within a pixel must be 
+stored contiguously in the linearized buffers. Non-empty fragments must be ordered
+front to back, and placed before any empty fragments. Layered-IceT internally 
+compresses the empty fragments within pixels for efficiency.
+
+The individually sorted pixels are then given in the same scanline
```

Comment:
Do we still need these lines @plhempel ? I find them a little difficult to understand

Reply:
Resolved in d00768be13abdaccb009a7bb2521759d877ce1f5